from django.apps import AppConfig


class CustomUsersConfig(AppConfig):
    default_auto_field = 'django.db.models.BigAutoField'
    name = 'django_exception_handling.custom_users'
